### DISCUSSION TITLE
Pro Only Integrations?

### DISCUSSION THREAD

**Discussion by frencojobs** [vercel/community#1466] [Help] [answered]
### Summary

Hi. I'm the guy who worked on a VSCode extension for Vercel about two years ago. I had to take a big break due to circumstances surrounding my country. I just started working on it again today and I had a really weird experience.

I was trying to replace sunsetted OAuth2 flow with new External mode, as I was prompted to do so, and I was able to test it just fine for a couple of attempts. Then suddenly the "Add Integration" button became "Pro Only Integration" button. Because I'm not an Enterprise user, I can't test it anymore.

<img width="508" alt="Screen Shot 2023-02-07 at 11 08 57 PM" src="https://user-images.githubusercontent.com/17830687/217314465-6da3dee1-8341-41ce-bc26-f130970edf49.png">

I also can't seem to be able to find more information about "Pro Only Integrations". Did I trigger some limits or something accidentally? I didn't even update the integration settings.

Would be great if I can get an explanation as to why it happened and I'd love to make it not "Pro Only" so that I can continue working on it. Thank you!

**Comment by amyegan** [comment]
Thanks for reaching out, @frencojobs. Can you share the link to the integration so I can investigate?

**Reply by frencojobs** [reply]
Sorry, it's https://vercel.com/integrations/vscode-vercel. If you go to `/new`, which is where I'm supposed to go for new external mode migration, you should be able to see the pro only button.

**Reply by kanamadiraju** [reply]
@amygen
Two days back I was able to access, But today suddenly got the same window shared by @frencojobs 
Tried with accessing the same by creating new user account still same issue persists
link to the integration: https://vercel.com/integrations/app-builder-console-v2/new?state=token=TOKEN

Any help please

**Reply by girishg4t** [reply]
Same issue started appearing for me as well from my morning, yesterday it was working fine.
I am not sure what change i need to make to get it working, @amyegan could you please help

**Reply by amyegan** [reply]
We are investigating this now. Thank you all for sharing these details 🙏

**Comment by amyegan** [comment, marked as answer]
The team found the source of the problem and released a fix! 🎉 Things should be back to normal now

**Reply by kanamadiraju** [reply]
@amyegan Thank you 😊 
Issue has been resolved and and It's working fine
